src/System.Private.Xml/src/project.json

```diff
@@ -27,7 +26,7 @@
         "System.IO.FileSystem.Primitives": "4.4.0-beta-24705-01",
         "System.Linq": "4.4.0-beta-24705-01",
         "System.Net.Http": "4.4.0-beta-24705-01",
-        "System.Net.Primitives": "4.4.0-beta-24705-01",
+        "System.Net.Requests": "4.4.0-beta-24705-01",
```

Comment:
I don't remember what type exactly was used from there, but it must've been in the code I removed with CodeDom, as looks like neither `System.Net.Requests` nor `System.Net.Primitives` are any longer needed. Will remove the dependency.